karate-tests/src/test/java/helpers/utils.feature

```diff
@@ -21,20 +21,17 @@ Scenario:
     return result;
   }
   """
-  * def randomBirthDate = 
+  * def getRandomBirthDate = 
   """
   function() {
     var min = new Date(Date.parse('1930-01-01')).getTime();
-    var max = new Date(Date.parse('2020-12-31')).getTime();
+    var max = new Date().getTime();
```

Comment:
I assume this returns today's date (as seconds since some date). This may cause issues if this date is within the last six months, as this would be rejected as the new record request should be done as a 'birth registration'.

Reply:
Ah good catch. I changed it based on a comment from Claudiu. So the latest date should be six months before today's date?

Reply:
Calculating the date 6 months before today isn't so trivial, and this is just test code that really just needs a valid date, so I propose just going back to something simple. I see no need to start writing lots of javascript just to "futureproof" this:

```js
    var min = new Date(Date.parse('1930-01-01')).getTime();
    var max = new Date(Date.parse('2023-09-01')).getTime();
```

Reply:
👍